leases/utils.py

```diff
@@ -43,6 +45,52 @@ def calculate_season_end_date(lease_end: date = None) -> date:
     return default
 
 
+def calculate_winter_season_start_date(lease_start: date = None) -> date:
+    """Return the date when the winter season starts
+
+    If the current date is after the season end date,
+    it returns next year's default start date.
+
+    Winter Storage Leases always start on 15.9 the earliest
+    """
+    today = date.today()
+    default = date(day=15, month=9, year=today.year)
+
+    if lease_start:
+        return default.replace(year=lease_start.year)
+
+    # If the current day is before the winter season ends,
+    # the start day will be on the previous year
+    if today < date(day=10, month=6, year=today.year):
+        default -= relativedelta(years=1)
+    # Otherwise, if the current day is after the season has started,
+    # the default start date will be on the following year
+    elif today >= date(day=15, month=9, year=today.year):
+        default += relativedelta(years=1)
```

Comment:
I don't thinkwe need this at all. 
- this function returns the start date for the current season, if season is going on right now
- it returns next season's startdate, if the season is already over

for this we need to:
- from 1.1 till 10.6 return 15.9 of the previous year
- from 10.6 return 15.9 of the current year

🤔

Reply:
so, if you just skip these lines, it should be ok, _I think_

Reply:
So on `10.10.2020` the start date would be `15.9.2020`, right? Then we can get rid of those lines.

Reply:
>So on 10.10.2020 the start date would be 15.9.2020, right? 

yes, I think that is the logic we need

Reply:
this will be consistent with the logic for summer seasons, where we also have these rules:
> - this function returns the start date for the current season, if season is going on right now
> - it returns next season's start date, if the season is already over

Reply:
Right, makes sense returning the beginning of the ongoing season. :thu